chart/kubecop/values.yaml

```diff
@@ -34,6 +34,12 @@ kubecop:
     enabled: true
     ignoreMounts: "true"
     ignorePrefixes: "/proc,/tmp"
+  clamAV:
+    enabled: false
+    host: "localhost"
+    port: 3310
+    path: "/host"
+    scanInterval: 600s # Scan interval in seconds.
```

Comment:
How much time takes a scan? Does ClamAV know if it already scanned file and second scans are faster?

Reply:
Let's raise it to be once an hour.
I don't know to tell if the second scans are faster yet...